nixos/modules/services/monitoring/grafana-alloy.nix

```diff
@@ -0,0 +1,80 @@
+{ lib, pkgs, config, ... }:
+with lib;
+let
+  cfg = config.services.grafana-alloy;
+in
+{
+  meta = {
+    maintainers = with maintainers; [ flokli ];
+  };
+
+  options.services.grafana-alloy = {
+    enable = mkEnableOption "grafana-alloy";
+
+    package = mkPackageOption pkgs "grafana-alloy" { };
+
+    configPath = mkOption {
+      type = lib.types.path;
+      default = "/etc/grafana-alloy";
```

Comment:
This should probably be /etc/alloy to keep in line with upstream packaging. Or specifically /etc/alloy/config.alloy

Reply:
Good point! I made this default to `/etc/alloy`, and updated the descriptions and `reloadTrigger` to suggest putting a `config.alloy` file in there.